ipaserver/install/bindinstance.py

```diff
@@ -775,6 +775,8 @@ def __setup_sub_dict(self):
             NAMED_PID=paths.NAMED_PID,
             NAMED_VAR_DIR=paths.NAMED_VAR_DIR,
             BIND_LDAP_SO=paths.BIND_LDAP_SO,
+            DATA_DIR='' if ipautil.file_exists(paths.ETC_DEBIAN_VERSION) else 'data/',
```

Comment:
You can use ipaplatform.NAME instead. It would also fix the pylint isues

```
import ipaplatform
...
DATA_DIR='' if ipaplatform.NAME == 'debian' else 'data/'
```